agent.py

```diff
@@ -54,6 +54,15 @@ def __init__(self, number:int, config: AgentConfig):
         self.agent_name = f"Agent {self.number}"
 
         self.system_prompt = files.read_file("./prompts/agent.system.md").replace("{", "{{").replace("}", "}}")
+        """
+        I'm adding this extra initialization prompt for the user.
+        The IRL user can add additional universal information that the agents
+        can always refer back to for operational instructions, but still remains 
+        outside the main system prompt. In my case, I want the agent to be able to
+        follow a guide I have in the work_dir, and I'll instruct the agent in this new md file
+        """
+        
+        self.user_initialization_prompt = files.read_file("./prompts/agent.user_initialization.md").replace("{", "{{").replace("}", "}}")
```

Comment:
I don't see this file anywhere. This will fail to read.

Reply:
Again, sorry for leaving the unfinished modification. I had intended on adding a kind of initialising library for the user. But I never followed through and ran out of time.